### PR TITLE
Merge 19.6 code freeze

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 1]
```diff
@@ -1,12 +1,4 @@
-We updated Comment Notifications with a spiffy new look and feel. (You might recognize it from My Site Comments!) The Quick Start design in your site menu and dashboard is rocking a new look, too.
+* [*] Enhances the exit animation of notices. [#18182]
+* [*] Media Permissions: display error message when using camera to capture photos and media permission not given [https://github.com/wordpress-mobile/WordPress-iOS/pull/18139]
+* [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [#18240]
```
Comment (mokagio) [inline WordPress/Jetpack/Resources/release_notes.txt:3]: @leandroalonso kindly gave a heads up that this final change applies to both WordPress and Jetpack. The result is that the release notes for the two apps are the same in this release, too.

Also worth noting that there are a few PRs targeting the 19.6 milestone that haven't been merged yet. I looked at them and they all either relate to the new site dashboard or are part of a WIP feature and so don't necessitate release notes yet.

File: WordPress/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -9485,6 +9615,9 @@
 /* This text is used when the user is configuring the iOS widget to suggest them to select the site to configure the widget for */
 "ios-widget.gpCwrM" = "Select Site";
 
+/* This text is required by Apple to be part of the translations but is not shown to the users. You can use the English version verbatim without translation for this entry. */
+"ios-widget.tVvJ9c" = "Select Site Intent";
```
Comment (mokagio) [inline WordPress/Resources/en.lproj/Localizable.strings:9619]: 👆 just a note here to confirm the automation added this entry. Not that there were doubts about it, but I didn't take it into account when I originally proposed this, which would have resulted in unnecessary noise in the Git log. See https://github.com/wordpress-mobile/WordPress-iOS/pull/18254#discussion_r840300107

**Review by mokagio — COMMENTED** [review]
- [x] Switch to production Pods
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `download_metadata.swift`

**Review by oguzkocer — APPROVED** [review]

**Comment by mokagio** [comment]
Thank you @antonis and @AliSoftware for the timely release notes update, and @oguzkocer for the review and merge ❤️ ❤️ ❤️

**Pull request merged**